<launch>
  <!-- 加载话题配置 -->
  <rosparam file="$(find arm_controller)/config/topics.yaml" command="load" />
  
  <node pkg="arm_controller" type="arm_controller_node.py" name="arm_controller_node" output="screen">
  </node>
</launch>